clusterloader2/pkg/measurement/common/scheduling_throughput.go

```diff
@@ -108,7 +99,7 @@ func (s *schedulingThroughputMeasurement) start(clientSet clientset.Interface, n
 
 	go func() {
 		defer ps.Stop()
-		selectorsString := measurementutil.CreateSelectorsString(namespace, labelSelector, fieldSelector)
+		selectorsString := selector.String()
```

Comment:
nit: Why is this variable needed? Can't you just pass selector to the klog directly? I don't see any other usages.

Reply:
I don't want to calculate every time. It will not change, so I just store it.

Reply:
That's three strings to concatenate. I don't think that would improve performance.

Reply:
It could visibly improve if we would be calling that often. But in this case we are not, so I agree with @oxddr that this isn't worth it.

Reply:
selectorString removed.